templates/step/edit.html.ep

```diff
@@ -83,18 +83,20 @@
                     % for my $tag (@$tags) {
                         <label>
                             <input type="checkbox" name="tags" id="tag_<%= $tag %>" class="tag_checkbox"
-                                   value="<%= $tag %>"><%= $tag %>
+                                   value="<%= $tag %>" <%= 'readonly="readonly"' unless (is_operator) %> ><%= $tag %>
                         </label><br/>
                     % }
                 </div>
-                <input id="newtag" style="width: calc(100% - 6px - 40px);" />
-                <input type="button" style="width: 34px;" class="button" value="Add" id="tag_add_button" />
+                % if (is_operator) {
+                    <input id="newtag" style="width: calc(100% - 6px - 40px);" />
+                    <input type="button" style="width: 34px;" class="button" value="Add" id="tag_add_button" />
```

Comment:
The same applies for radio buttons. But here I'm not sure if disabled is desired state. As one can use that to switch between different screenshots and matching areas just for screening purposes. But I would imagine changing that should not change (or appear to change) needle json.